src/integration/hubspotUpdatesController.ts

```diff
@@ -8,9 +8,14 @@ export class hubspotUpdatesController{
     private hubApiModel: HubspotApiModel;
     private saasApiModel: SaasquatchApiModel;
 
-    constructor(hApiKey: string, sApiKey: string, sTenantAlias: string){
+    // constructor(hApiKey: string, sApiKey: string, sTenantAlias: string){
+    //     this.saasApiModel = new SaasquatchApiModel(sApiKey, sTenantAlias);
+    //     this.hubApiModel = new HubspotApiModel(hApiKey);
+    // }
```

Comment:
Please remove commented code if possible.

Reply:
Sorry, I'm not sure why that is still showing up do you see the "Outdated" symbol? Perhaps you were looking at an older commit?

Reply:
My mistake I missed the outdated symbol.